PusherClient/Pusher.cs

```diff
@@ -133,6 +133,9 @@ public void Connect()
 
             UnregisterEventsOnConnection();
 
+            if (_connection != null)
+                _connection.Disconnect();
+

```

Comment:
I wasn't sure in which order they'd get merged in, and how quickly they'd both be merged in.